### ISSUE TITLE
`make dist` broken in linbox master

### ISSUE THREAD

**Issue by mkoeppe** [linbox-team/linbox#266]
This blocks the new CI workflow of FFLAS-FFPACK in the "dist" phase.
```
....
make  distdir-am
make[6]: *** No rule to make target 'test.matrix', needed by 'distdir-am'.  Stop.
```
(see https://github.com/linbox-team/fflas-ffpack/runs/2176689013?check_suite_focus=true)

**Comment by ClementPernet** [comment]
This should be fixed by 7bc9f04.

**Comment by mkoeppe** [comment]
Thanks!
